spec/helpers/models_helper.rb

```diff
@@ -0,0 +1,34 @@
+def create_and_activate_user(name)
+  user = User.create!(
+    name:, email: "#{name}@example.com", password: '123456', bio: "Bio of #{name}", photo: "#{name}.jpg"
```

Comment:
- [ ] You are creating a user without a name.

Reply:
When we are passing `name:name`, rubocop asks us to omit the value.

Reply:
> When we are passing `name:name`, rubocop asks us to omit the value.

@ahangarha, are the tests running at your end after omitting the values?

Reply:
Sure they pass.
![rspec0001](https://user-images.githubusercontent.com/11241315/173952016-3946b609-63a8-4322-90b4-4f5a6bf1c1df.png)

Reply:
When I added those omitted values, the tests ran at my end

Reply:
Check rubocop after that.

you should get something like this:
```
➜  BlogOnRails git:(integration-test) ✗ rubocop
Inspecting 36 files
........................C...........

Offenses:

spec/helpers/models_helper.rb:28:13: C: [Correctable] Style/HashSyntax: Omit the hash value.
      user: user,
            ^^^^
spec/helpers/models_helper.rb:29:13: C: [Correctable] Style/HashSyntax: Omit the hash value.
      post: post,
            ^^^^

36 files inspected, 2 offenses detected, 2 offenses autocorrectable
```

Reply:
> Sure they pass. ![rspec0001](https://user-images.githubusercontent.com/11241315/173952016-3946b609-63a8-4322-90b4-4f5a6bf1c1df.png)

- Please can you submit your project once more for review. 


It's strange because I took some time to make sure that the issue should not be on my end.

Reply:
Please make sure you have Ruby with version higher than 3.1

Reply:
Since Ruby 3.1.0,  we can omit hash value this way. If I don't, Rubocop gives error.

More info: https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/

![image](https://user-images.githubusercontent.com/11241315/173955363-694816d5-be19-4d27-b534-bc52304c9add.png)

I also didn't know this before I face with it

Reply:
@ahangarha, I see!. I'm still using 2.7.2 :smile: 

Thanks :pray: . I think this may be the issue.

Reply:
OMG :smile: 
That is why.

I am going to make another request